docs/LED-INDICATION.md

```diff
@@ -22,3 +22,16 @@ that deserve user attention.
 | 12  | Controller replied without TLS connection state. |
 | 13  | Controller replied without OCSP response. |
 | 14  | Failed to fetch or verify Controller certificate. |
+
+# Appstatus display using LEDs
+
+Starting from commit e31ec53c0dbec104e4818c998223449576585d6b, application status is displayed using LEDs.
```

Comment:
I don't think it is necessary to put here the information about when the feature was introduced. Better place for this would be a changelog (which we do not have). Or at least let's use the eve's next version number instead of the commit SHA.

Reply:
We haven't even done references to releases. Basically we assume that the information in the docs directories describe be system at the point those additions/changes were added to docs. Folks who want to look at old releases should pull the docs for those releases. This approach also handles deleting things; we'll delete them in the docs (as opposed to keeping text and saying that was removed in commit X release Y).

As everywhere the docs might lag the code but be that as it may.

So no need to have any reference to when it was introduced.